src/Modules/Settings/Dnn.PersonaBar.Security/Security.Web/src/components/auditCheck/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-danger */
```

Comment:
This is an unrelated change, but we needed that to fix that to have webpack compile successfully.